python/arcticdb/flattener.py

```diff
@@ -32,6 +35,10 @@ class Flattener:
 
     def __init__(self):
         self.custom_normalizer = get_custom_normalizer(False)
+        self.meta_structure_v2 = get_config_int("VersionStore.RecursiveNormalizerMetastructureV2") == 1
```

Comment:
I think it would be less confusing like this:
```suggestion
        self.meta_structure_v2 = get_config_int("VersionStore.RecursiveNormalizerMetastructure") == 2
```
Otherwise if we add a V3, how will it work?

Reply:
Sorry have missed earlier commetns. Amended